code/game/turfs/open/floor/plating.dm

```diff
@@ -152,3 +174,122 @@
 
 /turf/open/floor/plating/foam/tool_act(mob/living/user, obj/item/I, tool_type)
 	return
+
+/turf/open/floor/plating/r_plate
+	name = "reinforced plating"
+	desc = "Thick, tough flooring created with multiple layers of metal."
+	icon_state = "r_plate-0"
+
+	thermal_conductivity = 0.025
+	heat_capacity = INFINITY
+
+	baseturfs = /turf/open/floor/plating
+	allow_replacement = FALSE
+	RCD_proof = TRUE
+	upgradable = FALSE
+
+	var/d_state = PLATE_INTACT
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names